packages/app-utils/src/components/FontBootstrap.tsx

```diff
@@ -1,5 +1,11 @@
 import React, { createContext, useEffect, useState } from 'react';
-import 'fira';
+import '@fontsource/fira-mono/400.css';
+import '@fontsource/fira-sans/300.css';
+import '@fontsource/fira-sans/400.css';
+import '@fontsource/fira-sans/400-italic.css';
+import '@fontsource/fira-sans/500.css';
+import '@fontsource/fira-sans/600.css';
+import '@fontsource/fira-sans/700.css';
```

Comment:
```suggestion
import '@fontsource/fira-sans/300.css';
import '@fontsource/fira-sans/300-italic.css';
import '@fontsource/fira-sans/400.css';
import '@fontsource/fira-sans/400-italic.css';
import '@fontsource/fira-sans/500.css';
import '@fontsource/fira-sans/600.css';
import '@fontsource/fira-sans/700.css';
import '@fontsource/fira-sans/700-italic.css';
```

There's a couple places with 300 italic and 700 italic